grammar.pegjs

```diff
@@ -77,7 +77,7 @@ builtinCommandName
  = '[[' / '['
 
 argument "command argument"
- = commandName
+ = concatenation
```

Comment:
> In grammar.pegjs:
>
> > @@ -77,7 +77,7 @@ builtinCommandName
> >   = '[[' / '['
> >
> >  argument "command argument"
> > - = commandName
> > + = concatenation
>
> I believe keywords can be arguments. Not at my machine, but I think echo
done has no issues. Let me know if I'm misunderstanding you though.

I think the same, keywords can be used for arguments, since in bash they'll
only can appear where a command can.